### DISCUSSION TITLE
Ground Items - Contains or Wild Card Search

### DISCUSSION THREAD

**Discussion by LovetheDonut** [runelite/runelite#16633] [Ideas]
Not sure if this already exists and I haven't figured out how to use it yet.

If there was a way on ground items to filter ground items by a contains search or even a wild card search would be awesome.

For example, Grimy herbs.
At the moment, I've only been able to filter ground items by typing out the literal name of each herb.

If the search would allow for a partial search like Grimy], and filter any ground item with that literal at the beginning it would be awesome.

**Comment by YvesW** [comment]
``grimy*``?

**Reply by LovetheDonut** [reply]
my hero <3

**Comment by LovetheDonut** [comment]
Thanks to @YvesW for the answer.

Use an * at the end of your item.

Example:

grimy*
